### ISSUE TITLE
Represent the questions as JSON

### ISSUE THREAD

**Issue by Rittzz** [andykhov/Overwatch-Quiz#1]
To remove hardcoding, you can represent the question and answer as a JSON file using an asset or raw resource.  This has the nice side effect of setting up the infrastructure for pulling the JSON file from a remote source so the questions can be updated independent of recompiling the app.

This remote source could be just a static file publically hosted somewhere (github, pastebin, dropbox, etc.) or even a web service that returns a different set of questions on a per request basis!

**Comment by andykhov** [comment]
Oh, that's pretty interesting. So if I did want to add another question, I wouldn't have to "send out an update" if it was on the Play store?

Thanks for the tip!

**Comment by Rittzz** [comment]
Correct, you could then update the questions independently of updating the app on the play store which is a pretty big plus as issuing updates on the play store is quite time consuming.  In addition you aren't even guaranteed all your users would even update the app to get your new questions.

If you need any help just let me know.
